test/fs.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const metatests = require('metatests');
+const fs = require('fs');
+const { mkdirp } = require('..');
+const path = require('path');
+
+const beforeEach = dir => {
+  fs.access(dir, fs.constants.F_OK, err => {
+    if (err === null) {
+      fs.rmdir(dir, err => {
+        if (err) console.log(err);
+      });
+      fs.rmdir(path.dirname(dir), err => {
+        if (err) console.log(err);
+      });
+    }
+  });
+};
+
+const afterEach = dir => {
+  fs.rmdir(dir, err => {
+    if (err) console.log(err);
+    fs.rmdir(path.dirname(dir), err => {
+      if (err) console.log(err);
+    });
+  });
+};
+
+metatests.test('create 2 directories using mode', test => {
+  const dir1 = 'test/ex1/ex2';
+  beforeEach(dir1);
+  fs.access(dir1, fs.constants.F_OK, err => {
+    if (err === null || (err && err.code === 'ENOENT')) {
```

Comment:
`err === null`  means that `dir1` already exists which means that `mkdirp` won't do anything.